python/src/scipp/compat/mantid.py

```diff
@@ -706,7 +733,8 @@ def load(filename="",
          instrument_filename=None,
          error_connection=None,
          mantid_alg='Load',
-         mantid_args=None):
+         mantid_args=None,
+         advanced_geometry=False):
```

Comment:
Add documentation string for the new option?

Reply:
I'll merge this for now since it speeds up work significantly. Can you add the docstring in another PR?